src/Commands/EnvCreateCommand.php

```diff
@@ -56,8 +56,8 @@ public function createBuildEnv(
         // Revert to build:env:push if build:env:create is run against dev.
         if ('dev' === $multidev) {
             $this->log()->notice('dev has been passed to the multidev option. Reverting to dev:env:push as dev is not a multidev environment.');
-            // Run build:env:push.
-            $this->pushCodeToPantheon($site_env_id, $multidev, '', $env_label, $options['message'], $options['git-force']);
+            // Run build:env:push. Note the double-negative on no-git-force.
```

Comment:
`--no-git-force` is already in the docs. https://github.com/pantheon-systems/terminus-build-tools-plugin#command-options-5

I agree that the double negative is confusing. But at this point I think the easier fix is to change only this line to `no-git-force`

Reply:
As I said on the other issue, declaring `'git-force' => true` in $options creates a `--no-git-force` option that will set `$options['git-force']` to false if used.

The double-negative I was referring to was the variable used in pushCodeToPantheon.

If you define the option `'no-git-force'` instead of `'git-force'`, then the `--git-force` flag does not work. With the idiomatic solution, then you have both `--no-git-force` and `--git-force`.

So, in short, I still recommend the PR as currently written as the fix for this bug -- presuming that it works, of course.

Reply:
I'd go with the simpler change if that resolves the bug.

Reply:
I don't have time to iterate on this right now, but I have no objection if someone wants to take this in a different direction

Reply:
No worries, I'll give it a shot :)

Reply:
After looking into the annotated command library (thanks Greg!), I'm not sure that I like the double negative solution. It feels confusing. Do we really need a `--git-force` flag?

Reply:
Yes. Terminus Build Tools must use force so that it can overwrite the last commit with the build artifacts. Therefore, force must be the default. @stevector also has a use-case where he would rather not --force. Also, when we build the integration between build tools and integrated composer, we will need to support the no-force option.

I suppose our other option is to do a major version bump and say that the next version is IC only, and never force there. That's an easy solution, but I think we're going to want to maintain traditional build tools in parallel with the IC-supporting version, and it doesn't seem worth maintaining two separate versions just for the force flag.

I would still recommend that we solve this with the PR as I wrote it. Note that there is no double-negative in the use of the annotated flag. --git-force is true, and --no-git-force is false, as you would expect. The double-negative is only in the pushCodeToPantheon method. We could easily get rid of that by inverting the name and meaning and every usage of the variable name, and remove the ! in the call to the method as well. I held off on doing that due to lack of time and to keep the diff minimal so that we could just commit here and get the fix in.

It's up to you how to continue, but I don't think that the quick fix is bad; we can clean up the additional tech debt later without adding to it if we commit this now.